Code/Support/RKLumberjackLogger.m

```diff
@@ -75,7 +75,7 @@ + (void)logWithComponent:(_RKlcl_component_t)component
     [DDLog log:async
          level:componentLevel
           flag:flag
-       context:0 /* Could define a special value here to identify RestKit logs to any backend loggers */
+       context:0x524B5F00 + component
```

Comment:
I don’t think a constant is needed, but some documentation would be welcome. I searched for logging documentation but found nothing. Where would be an appropriate place to document this context value? README.md? (That’s what I did for [XCDYouTubeKit](https://github.com/0xced/XCDYouTubeKit/tree/d3732863e73d697afcf769f0ea55b44c9068e9d3#logging))

I also opened [issue #529](https://github.com/CocoaLumberjack/CocoaLumberjack/issues/529) on the CocoaLumberjack project.